File: Data Structures & Algorithms/merge-k-sorted-linked-lists/submission-0.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:    
    def mergeKLists(self, lists: List[Optional[ListNode]]) -> Optional[ListNode]:
        arr = [] 
        for list in lists :
            node = list
            while node:
                arr.append(node.val)
                node = node.next

            
        arr.sort()
        res = ListNode()
        curr = res
        for i in arr:
            curr.next = ListNode(i)
            curr = curr.next 
        
        return res.next 
                        

File: Data Structures & Algorithms/k-closest-points-to-origin/submission-0.cs
public class Solution {
    public int[][] KClosest(int[][] points, int k) {
        Array.Sort(points, (a, b)=> {
           long la = (long)(a[0]*a[0] + a[1]*a[1]);
           long lb = (long)(b[0]*b[0] + b[1]*b[1]);
           return la.CompareTo(lb) ;
        }) ; 

        var res = new int[k][];
        Array.Copy(points, res, k) ;
        return res ;
    }
}
